components/asset-metadata-service/pkg/signal/signal.go

```diff
@@ -0,0 +1,26 @@
+// Copied from github.com/kyma-project/kyma/components/binding-usage-controller/pkg/signal/signal.go
```

Comment:
Why copying instead of vendoring?

Reply:
Because it should belong to `commons` package.